### DISCUSSION TITLE
seata 和 sentinel 中的feignSentinelBuilder冲突

### DISCUSSION THREAD

**Discussion by liyf155** [apache/incubator-seata#4136] [Q&A] [answered]
***************************
APPLICATION FAILED TO START
***************************

Description:

The bean 'feignSentinelBuilder', defined in class path resource [com/alibaba/cloud/seata/feign/SeataFeignClientAutoConfiguration.class], could not be registered. A bean with that name has already been defined in class path resource [cn/examtec/examis/common/sentinel/SentinelAutoConfiguration.class] and overriding is disabled.

Action:

Consider renaming one of the beans or enabling overriding by setting spring.main.allow-bean-definition-overriding=true

Disconnected from the target VM, address: '127.0.0.1:64526', transport: 'socket'

Process finished with exit code 1

seata版本：1.4.2
sentinel版本：1.8.0

**Comment by funky-eyes** [comment, marked as answer]
反馈给spring-cloud-alibaba
